package.json

```diff
@@ -5,13 +5,17 @@
     "build": "webpack --config webpack.config.js"
```

Comment:
- [ ] Kindly fix this error
![image](https://github.com/Armynerh/rails_webpack_helloworld_app/assets/24703171/5a8c0e57-1ecd-46e9-9fe1-41387ff4999c)
I get this error when I run `npm run build` to build your project before running it.

Reply:
Well resolved! ✔️